Sources/Apollo/GraphQLGETTransformer.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  GraphQLGETTransformer.swift
+//  Apollo
+//
+//  Created by Ellen Shapiro on 7/1/19.
+//  Copyright © 2019 Apollo GraphQL. All rights reserved.
+//
+
+import Foundation
+
+struct GraphQLGETTransformer {
```

Comment:
Breaking this out and testing it is a great improvement. Just wondering why you decided to make this a struct instead of a top-level function. Were you thinking of extending this with additional methods or multi-stage building in the future?

Reply:
I prefer to make things named structs/classes/enums instead of free-floating functions. This is partly to make it easier to break things into additional methods in the future, but it's mostly for clarity of where the hell something is coming from.